package.json

```diff
@@ -54,7 +54,7 @@
     "react-icons": "^4.4.0",
     "react-redux": "^8.0.2",
     "uuid": "^8.3.2",
-    "wagmi": "^0.5.6"
+    "wagmi": "0.4.3"
```

Comment:
use latest wagmi like we do on everipedia pls

Reply:
Yes, I want to create a new PR to do that.